src/gamestate/dcon_generated.txt

```diff
@@ -1510,6 +1510,11 @@ object {
 		name{ subsistence_score }
 		type{ float }
 	}
+	property{
+		name{ rgo_was_set_during_scenario_creation }
+		type{ bitfield }
+		tag{ scenario, save }
```

Comment:
If this needs to be persistent, it should be in one or the other. But, since the initialize economy function is only run once, during scenario creation, it may not need to be persistent at all.